src/app_service/service/message/async_tasks.nim

```diff
@@ -1,6 +1,9 @@
 include ../../common/json_utils
 include ../../../app/core/tasks/common
 
+import status/statusgo_backend_new/chat as status_go_chat
```

Comment:
You cna technically import that in `message/service` instead since `async_taks` is `include`d

Reply:
Isn't it better to have import as close as possible to usage?

Reply:
Maybe. The `async_task` files have been made like I said so I just followed like a good sheep